src/cryptonote_core/miner.h

```diff
@@ -119,8 +139,39 @@ namespace cryptonote
     bool m_do_print_hashrate;
     bool m_do_mining;
 
-  };
-}
-
+    // background mining stuffs ..
 
+    static const int CPU_DATA_OVERALL_TIMES_COUNT = 4;
```

Comment:
Can these constants by in the cpp? Do they need to be exposed in the header for some reason?